### ISSUE TITLE
Parsing doesn't work.

### ISSUE THREAD

**Issue by qwerty70020** [vgavro/my-kyivstar#2]
my-kyivstar 8 <module>
sys.exit(main())

my_kyivstar.py 95 main
data = session.get_account_info()

my_kyivstar.py 63 get_account_info
data = self._parse_page_data(resp)

my_kyivstar.py 59 _parse_page_data
return json.loads(re.search('var pageData = ({.*});\n\n', resp.text).group(1))

AttributeError:
'NoneType' object has no attribute 'group'

**Comment by vgavro** [comment]
yeah, but even if you fix this problem - authentication still doesn't work, they changed their authentication logic.

The whole kyivstar login process is tricky and hard to debug, at the time I wrote this there was no reliable way to check if authentication was succeed or not.

Since some time they started to require password + SMS on web login, but this script somehow managed to login without SMS authentication. Now authentication even after parsing fix is not working - and reason is hard to debug.

For now only way to implement it in reliable way is to reverse engineer web browser authentication, but because of SMS required - there is not much interest in it for me. Also note that they also changed some endpoints.

If you want to work on this, here is parsing fix (but note - I don't even sure anymore that this variable is used):
```
-        bc = re.search('\[\'safari\'\], \'([\w\d]+)\'', resp.text).group(1)
+        bc = re.search(',bc=\'([\w\d]+)\'', resp.text).group(1)
```
